consultation_analyser/support_console/views.py

```diff
@@ -13,3 +18,34 @@ def support_home(request: HttpRequest):
 def sign_out(request: HttpRequest):
     logout(request)
     return redirect("/")
+
+
+@staff_member_required
+def show_consultations(request: HttpRequest) -> HttpResponse:
+    if request.POST:
+        try:
+            dummy_data.DummyConsultation(include_themes=False)
+            messages.add_message(request, messages.INFO, "New consultation generated")
```

Comment:
Here and below, can use the shorthand `messages.info` / `messages.warning`.

Reply:
I think messages don't behave quite as you're treating them here!

The process is:

view calls messages.add
page renders
on the _next_ request, the message is available to the view

the expected flow is usally

- receive POSTdata
- update a model
- set a message
- redirect 

(the classic post-and-redirect pattern)

so if you did have `messages` code in the templates, these messages would display one page _after_ when you wanted them to!

We don't have that code for now, tho, so we can fix this when we integrate messages in #107

Reply:
I might actually take the messages out of this PR and just add them properly afterwards (so we can test them properly etc.).

Reply:
Ok let's do that :+1: